### ISSUE TITLE
Region eu-west-2 (London) not supported

### ISSUE THREAD

**Issue by ivanaivana** [hashicorp/terraform#10726]
Please add support for the newly-added eu-west-2 (London) region:

```
$ terraform --version
Terraform v0.8.0

$ terraform plan
* Not a valid region: eu-west-2
```

**Comment by purejamie** [comment]
HI there - same here, we are very eager to get some infrastructure spun up in London so any updates appreciated.

**Comment by smayberry** [comment]
Support seems to be mostly in master already from #10470, but needs the Redshift account number from https://docs.aws.amazon.com/redshift/latest/mgmt/db-auditing.html#db-auditing-enable-logging

**Comment by stack72** [comment]
Hi @ivanaivana 

This was merged to master and will get released in a 0.8.1 soon™️ 

Thanks

Paul
